examples/azure-functions-timer-trigger/bal_new.out

```diff
@@ -0,0 +1 @@
+$ bal new azure-functions-timer-trigger
```

Comment:
Use underscores instead of dashes.

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-distribution/pull/4768.